content/blog/query-kubernetes/index.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Introducing Pulumi Query
+authors: ["joe-duffy"]
+tags: ["Pulumi-News", "Kubernetes"]
+meta_desc: "Today we announced Pulumi Pulumi Query, an SDK for programmatically querying cloud resources."
```

Comment:
"Announcing Pulumi Pulumi Query, an SDK for programmatically querying cloud resources."